packages/react-router-native/package.json

```diff
@@ -26,6 +27,8 @@
     "babel-preset-react-native": "1.9.1",
     "jest": "18.1.0",
     "react": "15.4.1",
+    "react-addons-test-utils": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
Some more things I've found.

The shallow renderer from react-addons-test-utils [may be extracted](https://github.com/facebook/react/issues/9207) from react-dom in React v16. 

---

Also, I dug way through the tree from react-test-renderer and found the important bits. I could extract this into a `while (!found) {}` loop, but I'm not sure if that would be any cleaner. At this point it is probably better to just grab enzyme for the nice API.

```jsx
import renderer from 'react-test-renderer'
const wrapper = renderer.create(<NativeRouter><Link to='/'><Text>text</Text></Link></NativeRouter>)
const output = wrapper._component._instance._reactInternalInstance._renderedComponent._instance._reactInternalInstance._renderedComponent._instance._reactInternalInstance._renderedComponent._instance._reactInternalInstance._renderedComponent._instance._reactInternalInstance._renderedComponent._currentElement
```

<details>
<summary>output</summary><p>

```jsx
{
  '$$typeof': Symbol(react.element),
  type: {
    [Function]
    displayName: 'TouchableHighlight',
  },
  key: null,
  ref: null,
  props: {
    to: '/',
    children: {
      '$$typeof': Symbol(react.element),
      type: [Function: Component],
      key: null,
      ref: null,
      props: [Object],
      _owner: null,
      _store: {}
    },
    replace: false,
    onPress: [Function],
    activeOpacity: 0.85,
    underlayColor: 'black'
  }
}
```

</p></details>